.meteor/packages

```diff
@@ -56,3 +55,6 @@ kadira:flow-router
 scorpiusjs:config
 scorpiusjs:materialize-admin-only
 useraccounts:bootstrap
+alethes:pages
+kurounin:pagination
+kurounin:pagination-blaze
```

Comment:
Reinventing the wheel seems to be a solution to me.
P.S. Release Candidate 4 of 1.5 is out as of writing.